app/models/comment.rb

```diff
@@ -1,4 +1,5 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+    scope :update_comments_counter, ->(post, count) { post.update(comments_counter: count) }
```

Comment:
- [ ] According to the guidelines you should have a method that returns the 5 most recent comments for a given post. You can check out the guide [Scope vs Class Methods](https://www.square63.com/scope-vs-class-methods/)

```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```